bot/start.ts

```diff
@@ -327,6 +327,8 @@ const initialize = (botToken: string, options: Partial<Telegraf.Options<MainCont
       order.canceled_by = ctx.admin._id;
       const buyer = await User.findOne({ _id: order.buyer_id });
       const seller = await User.findOne({ _id: order.seller_id });
+      if (buyer === null) return;
+      if (seller === null) return;
```

Comment:
@Mersho why are you changing behaviour here?

Reply:
> @Mersho why are you changing behaviour here?

@knocte On lines 334 and 336, we received an error because buyer & seller might be null.

Reply:
what error? you mean compiler error?

Reply:
I'm combining these two ifs

`if (buyer === null || seller === null) return;`

Reply:
you didn't answer my last question

Reply:
> what error? you mean compiler error?

yes, this is the error msg:

```
Argument of type '(UserDocument & { _id: ObjectId; }) | null' is not assignable to parameter of type 'UserDocument'.
  Type 'null' is not assignable to type 'UserDocument'.
```